packages/modules/stock-location/src/migrations/Migration20241206092214.ts

```diff
@@ -0,0 +1,13 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20241206092214 extends Migration {
+
+  async up(): Promise<void> {
+    this.addSql('alter table if exists "stock_location" add constraint "stock_location_address_id_unique" unique ("address_id");');
```

Comment:
Do we want to make this unique?
In this case, shouldn't it be WHERE on deleted_at is not null?

Reply:
I think this uniqueness is added by the DML `FK` thing. But yeah, we have to revisit it

Reply:
Is this something we should address now? In any case, let's create a ticket for it if we don't have it

Reply:
Fixed. It was because of the way relationship was setup.